assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceTilesWidget/AudienceTiles.js

```diff
@@ -50,6 +50,9 @@ import {
 import AudienceTile from './AudienceTile';
 import InfoTooltip from '../../../../../../components/InfoTooltip';
 import AudienceTooltipMessage from './AudienceTooltipMessage';
+import PlaceholderTile from './PlaceholderTile';
+import { Fragment } from 'react';
```

Comment:
This should be imported from `@wordpress/element`.

Reply:
D'oh :facepalm: 

Thanks, fixed :)